Checkers/Board.cs

```diff
@@ -0,0 +1,272 @@
+        public void PlaceCheckers()
+        {
+            for (var i = 0; i < Checkers.Count; i++)
+            {
+                int[] position = Checkers[i].Position;
+                this.Grid[position[0]][position[1]] = Checkers[i].Symbol;
+            }
+        }
+
+
+        
+        
+        //draws the board based on position of checkers with a virtual board from a jagged array
+        public void DrawBoard()
+        {
+            CreateBoard();
+            GenerateCheckers();
+            PlaceCheckers();
+            
+
+            for(int i = 0 ; i < 8 ; i++)
+            {
+                Console.Write(i.ToString());
+                for(int j = 0; j < 8; j++)
+                {
+                    Console.Write( "|" + this.Grid[i][j]);
+                }
+                Console.Write("|");
+                Console.WriteLine("\r");              
+
+            }
+            
+        }
+
+        // used to refresh board after moves/removes
+        public void reloadBoard()
+        {
+            CreateBoard();
+            PlaceCheckers();
+            for(int i = 0 ; i < 8 ; i++)
+            {
+                Console.Write(i.ToString());
+                for(int j = 0; j < 8; j++)
+                {
+                    Console.Write( "|" + this.Grid[i][j]);
+                }
+                Console.Write("|");
+                Console.WriteLine("\r");
+                
+            }
+
+        }
+        
+        // used to return checker class object found based on coordinates
+        public Checker SelectChecker(int row, int column)
+        {
+            return Checkers.Find(x => x.Position.SequenceEqual(new List<int> { row, column }));
+        }
+
+
+        // main method invoked for input and moving checkers no board
+        public void moveCheckerOnBoard()
+        {
+
+            int oldRow = 0;
+            int oldColumn = 0;
+            int newRow = 0;
+            int newColumn = 0;
+
+            Console.WriteLine("Enter pick up row.");
+            oldRow = int.Parse(Console.ReadLine());
+            Console.WriteLine("Enter pick up column.");
+            oldColumn = int.Parse(Console.ReadLine());
+
+            Console.WriteLine("Enter placement row.");
+            newRow = int.Parse(Console.ReadLine());
+            Console.WriteLine("Enter placement column.");
+            newColumn = int.Parse(Console.ReadLine());
+
+            moveChecker(oldRow, oldColumn, newRow, newColumn);
+            reloadBoard();
+            
+
+        }
+
+        // method called to move checker from prior position
+        public void moveChecker(int oldR, int oldC, int newR, int newC)
+        {
+            int oldRow = oldR;
+            int oldColumn = oldC;
+            int newRow = newR;
+            int newColumn = newC;
+            Checker checkerMoved ;
+            Checker placedChecker ;
+
+
+            checkerMoved = SelectChecker(oldRow, oldColumn);
```

Comment:
Make sure you test adding selecting a checker at an invalid location. For example row 99, column -1.